variables.tf

```diff
@@ -271,6 +284,28 @@ EOF
 
 }
 
+variable "http_proxy" {
+  type    = string
+  default = ""
+
+  description = <<EOF
+HTTP proxy for downloading agents and vault executable on startup. Only necessary if allow_public_egress is false.
+This is only used on the first startup of the Vault cluster and will NOT set the global HTTP_PROXY environment variable.
+i.e. If you configure Vault to manage credentials for other services, default HTTP routes will be taken.
+EOF
+}
+
+variable "load_balancing_scheme" {
+  type    = string
+  default = "EXTERNAL"
+
+  description = <<EOF
+Options are INTERNAL or EXTERNAL.
```

Comment:
Not prod ready yet, but https://www.terraform.io/docs/configuration/variables.html#custom-validation-rules could be useful to revisit in the future.

Reply:
Great idea, I'll add this link in a comment to look at in the future.